contributors/devel/sig-testing/testing.md

```diff
@@ -48,17 +48,6 @@ passing, so it is often a good idea to make sure the e2e tests work as well.
 cd kubernetes
 make test  # Run all unit tests.
 ```
-If you have `GOPATH` set up correctly, you can
-also just use `go test` directly.
-
-```sh
-cd kubernetes
-go test ./... # Run all unit tests
-```
-
-The remainder of this documentation presumes that you use `Make` as an
-entry point, but remember that the ability to use `go test` exists should you
-desire.
```

Comment:
Except it didn't cause more examples got added, haha. ```the ability to use `go test` exists should you
desire.``` is really the overall lesson of this.